src/app/components/paragraph/paragraph.component.html

```diff
@@ -0,0 +1,7 @@
+<p>
+	<span class="p-num">{{data.n}}.</span>
+	<evt-content-viewer class="genericElement {{data.class}}" *ngFor="let el of data.content" [content]="el"
```

Comment:
Why the "genericElement" class?

Reply:
Copy-Paste error from generic-component in here too! 🙄
Also, i'm going to move the class attribute since `{{data.class}}` refers to the `<p>` tagname.

Reply:
We can probably drop the class attribute in this case, since it will not make much sense to have `<p class="p">` (knowing that the class is the tag name of the original `<p>` in the xml)